NEXT_CHANGELOG.md

```diff
@@ -31,6 +48,23 @@ Callers may need to add either a operator `?` (in plugins) or an `.unwrap()` cal
 
 By [@SimonSapin](https://github.com/SimonSapin)
 
+### rename originating_request to supergraph_request on various plugin Request structures ([Issue #1713](https://github.com/apollographql/router/issues/1713))
```

Comment:
Presentation consistencies that I think "look nice" in the final rendered shipped release notes:

```suggestion
### Rename `originating_request` to `supergraph_request` on various plugin `Request` structures ([Issue #1713](https://github.com/apollographql/router/issues/1713))
```